docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+
+  proxy:
+    build: ./
+    # depends_on:
+    #   - 'carousel'
```

Comment:
I would probably add depends_on: carousel and checkout here. You want these two apps to be loaded before your proxy server can reach out to them.